data/ill_requests.json

```diff
@@ -21,7 +21,7 @@
       "title": "L'invention du pain",
       "year": "1920",
       "authors": "Erken Beno\u00eet",
-      "published_in": {
+      "source": {
```

Comment:
Nope. `source` != `published_in`.

`published_in` is, for instance, the journal in which the requested article is published, hence `published_in`.

`found_in` is where the requested document has been found, and `found_in.source` is the name of the catalog.

Reply:
according to json_schema, using `source` seems correct for these fixtures